extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -20,26 +20,38 @@ import {
 } from './amp-story-interactive-abstract';
 import {CSS} from '../../../build/amp-story-interactive-slider-0.1.css';
 import {htmlFor} from '#core/dom/static-template';
+//import {scopedQuerySelector, scopedQuerySelectorAll} from '#core/dom/query';
+import {setImportantStyles} from '#core/dom/style';
 
 /**
  * Generates the template for the slider.
  *
  * @param {!Element} element
  * @return {!Element}
  */
+
 const buildSliderTemplate = (element) => {
   const html = htmlFor(element);
   return html`
     <div class="i-amphtml-story-interactive-slider-container">
       <div class="i-amphtml-story-interactive-prompt-container"></div>
       <div class="i-amphtml-story-interactive-slider-input-container">
-        <input
-          class="i-amphtml-story-interactive-slider-input"
-          type="range"
-          min="0"
-          max="100"
-          value="25"
-        />
+        <div class="i-amphtml-story-interactive-slider-field">
+          <div class="i-amphtml-story-interactive-side-values">0</div>
+          <div class="i-amphtml-story-interactive-slider-input-size">
+            <input
+              class="i-amphtml-story-interactive-slider-input"
+              type="range"
+              min="0"
+              max="100"
+              value="0"
+            />
+            <div class="i-amphtml-story-interactive-slider-bubble span">
```

Comment:
This can be just `i-amphtml-story-interactive-slider-bubble` without the `span`.

When targeting in CSS, writting `.i-amphtml-story-interactive-slider-bubble span` will target the span element inside the element with the `i-amphtml-story-interactive-slider-bubble` class :) 

[This game](https://flukeout.github.io/) is pretty fun for learning about selectors.